incubator/hnc/README.md

```diff
@@ -58,6 +58,20 @@ scripts](https://docs.google.com/document/d/1tKQgtMSf0wfT3NOGQx9ExUQ-B8UkkdVZB6m
 to get an idea of what HNC can do. For a more in-depth understanding, check out
 the [HNC Concepts doc](http://bit.ly/38YYhE0).
 
+### Viewing metrics
+You should be able to view all HNC metrics in your perferred backend:
+* Stackdriver on GKE (see [instructions](https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/doc/metrics/stackdriver-gke.md))
```

Comment:
I'd just make "Stackdriver on GKE" the link

Reply:
Also I _think_ the link can just be (stackdriver-gke.md) since it's a link to a file in the same directory. This also means that if we move the code, we don't have to update all the links.

Reply:
SG. I used "doc/metrics/stackdriver-gke.md".